tests/unittests/tensorsTest.cpp

```diff
@@ -399,6 +399,34 @@ TEST(Tensor, nonOwnedTensor) {
   H1.dump();
 }
 
+/// Check that we properly take ownership of
+/// the underlying memory when we reset the tensor
+/// shape. This test used to fail leak sanitizer.
+TEST(Tensor, nonOwnedTensorFollowedByReset) {
+  uint8_t raw_data[] = {0, 0, 0, 0};
+  Type F32Ty(ElemKind::FloatTy, {1});
+
+  // Create an unowned tensor.
+  Tensor T1(raw_data, &F32Ty);
```

Comment:
nit: Since this is `FloatTy`, why aren't you just creating the raw_data as `float raw_data[] = {0.};`?

Reply:
Good point! I guess I was too attached as representing the memory byte by byte :P